MovieNight/MovieNightApiClient.swift

```diff
@@ -57,12 +57,9 @@ enum MovieNightEndpoint: Endpoint {
     var parameters: [String : String] {
         var parameters = [String : String]()
         switch self {
-        case .Actor(let page), .Movie(let page) :
-            parameters["page"] = page
-            parameters["api_key"] = "57b213eb9d700e45c3f1ddaa754d7134"
-            return parameters
-        default:
+        case .Actor(let page), .Movie(let page), .Genre(let page) :
```

Comment:
If the parameters are the same for all the enum types, then you don't need the switch.